.travis.yml

```diff
@@ -21,58 +18,40 @@ script:
       else
         ./configure $ARG && make -j2 V=0;
       fi
-    - if [ "$VALGRIND" = "true" ]; then
-        make run-valgrind V=0;
-      elif [ "$BUILD_TYPE" != "ios" ]; then
+    - if [ "$BUILD_TYPE" != "ios" ]; then
         make -j2 check-am V=0;
       fi
 
 # Silence gcov standard output since it produces way too much output
 after_success:
     - if [ "$COVERAGE" = "true" ]; then
-        coveralls --gcov /usr/bin/gcov-4.6 --exclude src/ext
-                  --exclude example --exclude third_party > /dev/null;
+        coveralls --gcov /usr/bin/gcov-5 --exclude src/ext -b .
```

Comment:
Adding `-b .` fixed running `gcov` for `gcc`, which was previously not possible.